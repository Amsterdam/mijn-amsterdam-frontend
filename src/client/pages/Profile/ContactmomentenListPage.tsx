import { contactmomentenDisplayProps } from './config';
import { useContactmomenten } from './useContactmomenten.hook';
import { AppRoutes } from '../../../universal/config/routes';
import { Themas } from '../../../universal/config/thema';
import { ListPagePaginated } from '../../components/ListPagePaginated/ListPagePaginated';

export function ContactmomentenListPage() {
  const { contactmomenten, isLoading, isError, title } = useContactmomenten();

  return (
    <ListPagePaginated
      items={contactmomenten}
      title={`Alle ${title.toLowerCase()}`}
      appRoute={AppRoutes['KLANT_CONTACT/CONTACTMOMENTEN']}
      appRouteBack={AppRoutes.BRP}
      displayProps={contactmomentenDisplayProps}
      thema={Themas.BRP}
      isLoading={isLoading}
      isError={isError}
    />
  );
}
