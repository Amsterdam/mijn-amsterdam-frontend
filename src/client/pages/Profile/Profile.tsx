import {
  Alert,
  ChapterIcon,
  DetailPage,
  InfoPanel,
  InfoPanelCollapsible,
  LinkdInline,
  LoadingContent,
  PageContent,
  PageHeading,
} from '../../components';
import { PanelConfigFormatter, panelConfig } from './Profile.constants';
import React, { useContext, useMemo } from 'react';

import { AppContext } from '../../AppState';
import { BrpApiState } from '../../hooks/api/api.brp';
import { defaultDateFormat } from '../../../universal/helpers';
import { formatBrpProfileData } from '../../data-formatting/brp';
import styles from './Profile.module.scss';

function formatInfoPanelConfig(
  panelConfig: PanelConfigFormatter,
  BRP: BrpApiState
) {
  if (typeof panelConfig === 'function') {
    return panelConfig(BRP);
  }
  return panelConfig;
}

export default function Profile() {
  const { BRP } = useContext(AppContext);

  const { isDirty, isError, data } = BRP;
  const brpProfileData = useMemo(() => {
    const rData = isDirty && !isError ? data : null;
    return rData ? formatBrpProfileData(rData) : rData;
  }, [data, isDirty, isError]);

  return (
    <DetailPage className={styles.Profile}>
      <PageHeading icon={<ChapterIcon />}>Mijn gegevens</PageHeading>
      <PageContent className={styles.Intro}>
        <p>
          In de Basisregistratie Personen legt de gemeente persoonsgegevens over
          u vast. Het gaat hier bijvoorbeeld om uw naam, adres, geboortedatum of
          uw burgerlijke staat. De gemeente gebruikt deze gegevens. Belangrijk
          dus dat deze gegevens kloppen.
        </p>

        {BRP.isLoading && (
          <div className={styles.LoadingContent}>
            <LoadingContent />
            <LoadingContent />
            <LoadingContent />
          </div>
        )}

        {BRP.isError && (
          <Alert type="warning">
            <p>We kunnen op dit moment geen gegevens tonen.</p>
          </Alert>
        )}

        {data.persoon?.vertrokkenOnbekendWaarheen && (
          <Alert type="warning" className="vertrokkenOnbekendWaarheen">
            <p>
              U staat sinds{' '}
              {data.persoon.datumVertrekUitNederland
                ? defaultDateFormat(data.persoon.datumVertrekUitNederland)
                : 'enige tijd'}{' '}
              in de BRP geregistreerd als "vertrokken â€“ onbekend waarheen".
            </p>
            <p>
              U kunt uw huidige adres doorgeven bij het Stadsloket. U moet
              hiervoor een{' '}
              <LinkdInline
                external={true}
                href="https://www.amsterdam.nl/veelgevraagd/?productid=%7BCAE578D9-A593-40FC-97C6-46BEA5B51319%7D"
              >
                afspraak
              </LinkdInline>{' '}
              maken .
            </p>
          </Alert>
        )}

        {data?.adres?.inOnderzoek && (
          <Alert type="warning" className="inOnderzoek">
            <p>
              Op dit moment onderzoeken wij of u nog steeds woont op het adres
              waar u ingeschreven staat. Kijk voor{' '}
              <LinkdInline
                external={true}
                href="https://www.amsterdam.nl/veelgevraagd/?productid={49AB6693-E7FA-4642-82F4-D14D87E02C72}"
              >
                meer
              </LinkdInline>{' '}
              informatie over onderzoek naar uw inschrijving in de
              Basisregistratie Personen (BRP).
            </p>
            <p>
              Kloppen uw gegevens niet? Voorkom een boete en stuur een bericht
              naar{' '}
              <a
                href="mailto:adresonderzoek.basisinformatie@amsterdam.nl"
                rel="external noopener noreferrer"
              >
                adresonderzoek.basisinformatie@amsterdam.nl
              </a>
              .
            </p>
          </Alert>
        )}
      </PageContent>

      {!!brpProfileData?.persoon && (
        <InfoPanel
          className={styles.DefaultPanel}
          {...formatInfoPanelConfig(panelConfig.persoon, BRP)}
          panelData={brpProfileData.persoon}
        />
      )}

      {!!brpProfileData?.adres && (
        <InfoPanel
          className={styles.DefaultPanel}
          {...formatInfoPanelConfig(panelConfig.adres, BRP)}
          panelData={brpProfileData.adres}
        />
      )}

      {!!brpProfileData?.verbintenis && (
        <InfoPanelCollapsible
          id="profile-verbintenis"
          className={styles.Verbintenis}
          {...formatInfoPanelConfig(panelConfig.verbintenis, BRP)}
          panelData={brpProfileData.verbintenis}
        />
      )}

      {!!brpProfileData?.verbintenisHistorisch &&
        brpProfileData?.verbintenisHistorisch.length && (
          <InfoPanelCollapsible
            id="profile-verbintenisHistorisch"
            className={styles.Verbintenis}
            {...formatInfoPanelConfig(panelConfig.verbintenisHistorisch, BRP)}
            panelData={brpProfileData.verbintenisHistorisch}
          />
        )}

      {!!brpProfileData?.kinderen && brpProfileData.kinderen.length && (
        <InfoPanelCollapsible
          id="profile-kinderen"
          {...formatInfoPanelConfig(panelConfig.kinderen, BRP)}
          panelData={brpProfileData.kinderen}
        />
      )}

      {!!brpProfileData?.ouders && brpProfileData.ouders.length && (
        <InfoPanelCollapsible
          id="profile-ouders"
          {...formatInfoPanelConfig(panelConfig.ouders, BRP)}
          panelData={brpProfileData.ouders}
        />
      )}

      {!!brpProfileData?.adresHistorisch &&
        brpProfileData?.adresHistorisch.length && (
          <InfoPanelCollapsible
            id="profile-adresHistorisch"
            {...formatInfoPanelConfig(panelConfig.adresHistorisch, BRP)}
            panelData={brpProfileData.adresHistorisch}
          />
        )}
    </DetailPage>
  );
}
