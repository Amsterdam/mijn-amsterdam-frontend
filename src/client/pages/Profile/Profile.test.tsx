import AppState, { AppState as AppStateInterface } from '../../AppState';
import { mount, shallow } from 'enzyme';

import { BrowserRouter } from 'react-router-dom';
import Profile from './Profile';
import React from 'react';
import { ReactNode } from 'react';

const responseData = {
  adres: {
    inOnderzoek: true,
    huisletter: null,
    huisnummer: '1',
    huisnummertoevoeging: null,
    postcode: '1064 BH',
    straatnaam: 'Burgemeester R\u00f6ellstr',
    woonplaatsNaam: 'Amsterdam',
    begindatumVerblijf: '1967-01-01T00:00:00Z',
  },
  adresHistorisch: [
    {
      inOnderzoek: true,
      huisletter: null,
      huisnummer: '1',
      huisnummertoevoeging: null,
      postcode: '1064 BH',
      straatnaam: 'Burgemeester R\u00f6ellstr',
      woonplaatsNaam: 'Amsterdam',
      begindatumVerblijf: '1967-01-01T00:00:00Z',
    },
    {
      inOnderzoek: true,
      huisletter: null,
      huisnummer: '1',
      huisnummertoevoeging: null,
      postcode: '1064 BH',
      straatnaam: 'Burgemeester R\u00f6ellstr',
      woonplaatsNaam: 'Amsterdam',
      begindatumVerblijf: '1967-01-01T00:00:00Z',
    },
  ],
  persoon: {
    vertrokkenOnbekendWaarheen: true,
    datumVertrekUitNederland: '1967-01-01T00:00:00Z',
    aanduidingNaamgebruikOmschrijving: 'Eigen geslachtsnaam',
    omschrijvingAdellijkeTitel: 'Hertogin',
    bsn: '129095205',
    geboortedatum: '1950-01-01T00:00:00Z',
    geboortelandnaam: 'Nederland',
    geboorteplaatsnaam: 'Lochem',
    gemeentenaamInschrijving: 'Amsterdam',
    geslachtsnaam: 'Beemsterboer',
    nationaliteiten: [
      {
        omschrijving: 'Nederlandse',
      },
      {
        omschrijving: 'Turkse',
      },
    ],
    omschrijvingBurgerlijkeStaat: 'Gehuwd',
    omschrijvingGeslachtsaanduiding: 'Man',
    opgemaakteNaam: 'W. Beemsterboer',
    voornamen: 'Wesley',
    voorvoegselGeslachtsnaam: null,
    mokum: true,
  },
  ouders: [
    {
      geboortedatum: '1920-01-01T00:00:00Z',
      geboortelandnaam: 'Nederland',
      geboorteplaatsnaam: 'Lochem',
      geslachtsnaam: 'Beemsterboer',
      nationaliteiten: [
        {
          omschrijving: 'Nederlandse',
        },
      ],
      omschrijvingGeslachtsaanduiding: 'Man',
      opgemaakteNaam: 'S. Beemsterboer',
      voornamen: 'Senior',
      bsn: '123456780',
      voorvoegselGeslachtsnaam: null,
    },
    {
      geboortedatum: '1920-01-01T00:00:00Z',
      geboortelandnaam: 'Nederland',
      geboorteplaatsnaam: 'Lochem',
      geslachtsnaam: 'Beemsterboerin',
      bsn: '123456780',
      nationaliteiten: [
        {
          omschrijving: 'Nederlandse',
        },
      ],
      omschrijvingGeslachtsaanduiding: 'Vrouw',
      opgemaakteNaam: 'Sa. Beemsterboer',
      voornamen: 'Seniora',
      voorvoegselGeslachtsnaam: null,
    },
  ],
  verbintenis: {
    datumOntbinding: null,
    datumSluiting: '1999-01-01T01:01:01.000Z',
    landnaamSluiting: 'Marokko',
    persoon: {
      bsn: '123456780',
      geboortedatum: '2019-07-08T09:14:58.963Z',
      geslachtsaanduiding: null,
      geslachtsnaam: 'Bakker',
      overlijdensdatum: null,
      voornamen: 'Souad',
      voorvoegselGeslachtsnaam: null,
    },
    plaatsnaamSluitingOmschrijving: 'Asilah',
    soortVerbintenis: 'H',
    soortVerbintenisOmschrijving: 'Huwelijk',
  },
  verbintenisHistorisch: [
    {
      datumOntbinding: '1998-07-08T09:14:58.963Z',
      datumSluiting: '1912-01-01T01:01:01.000Z',
      landnaamSluiting: 'Marokko',
      persoon: {
        bsn: '123456780',
        geboortedatum: '2019-07-08T09:14:58.963Z',
        geslachtsaanduiding: null,
        geslachtsnaam: 'Bakker',
        overlijdensdatum: null,
        voornamen: 'Souad',
        voorvoegselGeslachtsnaam: null,
      },
      plaatsnaamSluitingOmschrijving: 'Asilah',
      soortVerbintenis: 'H',
      soortVerbintenisOmschrijving: 'Huwelijk',
    },
    {
      datumOntbinding: '2019-07-08T09:14:58.963Z',
      datumSluiting: '',
      landnaamSluiting: 'Marokko',
      persoon: {
        bsn: '123456780',
        geboortedatum: '2019-07-08T09:14:58.963Z',
        geslachtsaanduiding: null,
        geslachtsnaam: 'Bakker',
        overlijdensdatum: null,
        voornamen: 'Souad',
        voorvoegselGeslachtsnaam: null,
      },
      plaatsnaamSluitingOmschrijving: 'Asilah',
      soortVerbintenis: 'H',
      soortVerbintenisOmschrijving: 'Huwelijk',
    },
  ],
  kinderen: [
    {
      bsn: '123456780',
      geboortedatum: '2019-07-08T09:14:58.963Z',
      geslachtsaanduiding: 'M',
      geslachtsnaam: 'Kosterijk',
      overlijdensdatum: null,
      voornamen: 'Yassine',
      voorvoegselGeslachtsnaam: null,
    },
    {
      bsn: '123456780',
      geboortedatum: '2019-07-08T09:14:58.963Z',
      geslachtsaanduiding: 'M',
      geslachtsnaam: 'Kosterijk',
      overlijdensdatum: '2019-07-08T09:14:58.963Z',
      voornamen: 'Marwan',
      voorvoegselGeslachtsnaam: null,
    },
  ],
};

function getAppState() {
  return {
    BRP: {
      isDirty: true,
      isLoading: false,
      isPristine: false,
      isError: false,
      data: JSON.parse(JSON.stringify(responseData)),
    },
  } as AppStateInterface;
}

function mountWithAppstate(appState: AppStateInterface, component: ReactNode) {
  return mount(
    <BrowserRouter>
      <AppState value={appState as AppStateInterface}>{component}</AppState>
    </BrowserRouter>
  );
}

describe('BRP Profile page', () => {
  beforeAll(() => {
    (window.matchMedia as any) = jest.fn(() => {
      return {
        addListener: jest.fn(),
        removeListener: jest.fn(),
      };
    });
  });

  it('Renders without crashing', () => {
    shallow(
      <AppState value={getAppState()}>
        <Profile />
      </AppState>
    );
  });

  it('Renders all the information provided', () => {
    const appState = getAppState();
    const page = mountWithAppstate(appState, <Profile />);

    expect(page.html()).toMatchSnapshot();
  });

  it('Does display verbintenis information if present', () => {
    const appState = getAppState();
    const page = mountWithAppstate(appState, <Profile />);

    expect(page.find(`.InfoPanel.Verbintenis`)).toHaveLength(2);
  });

  it('Does not display verbintenis information if not present', () => {
    const appState = getAppState();
    delete appState.BRP.data.verbintenis;
    delete appState.BRP.data.verbintenisHistorisch;

    const page = mountWithAppstate(appState, <Profile />);

    expect(page.find(`.InfoPanel.Verbintenis`)).toHaveLength(0);
  });

  it('Does not display country and place of birth when NOT a resident of Amsterdam', () => {
    const appState = getAppState();
    appState.BRP.data.persoon.mokum = false;
    appState.BRP.data.persoon.geboorteplaatsnaam = '';
    appState.BRP.data.persoon.geboortelandnaam = '';

    const page = mountWithAppstate(appState, <Profile />);

    expect(
      page
        .find('.InfoPanel')
        .at(0)
        .find(`.InfoPanelTableRow__geboorteplaats td`)
    ).toHaveLength(0);
    expect(
      page
        .find('.InfoPanel')
        .at(0)
        .find(`.InfoPanelTableRow__geboorteland td`)
    ).toHaveLength(0);
  });

  it('Displays onbekend as value for country and place of birth when IS resident of Amsterdam', () => {
    const appState = getAppState();
    appState.BRP.data.persoon.geboorteplaatsnaam = '';
    appState.BRP.data.persoon.geboortelandnaam = '';

    const page = mountWithAppstate(appState, <Profile />);

    expect(
      page
        .find('.InfoPanel')
        .at(0)
        .find(`.InfoPanelTableRow__geboorteplaats td`)
        .text()
    ).toBe('Onbekend');
    expect(
      page
        .find('.InfoPanel')
        .at(0)
        .find(`.InfoPanelTableRow__geboorteland td`)
        .text()
    ).toBe('Onbekend');
  });

  it('Displays an alert when adres.inOnderzoek and/or persoon.vertrokkenOnbekendWaarheen is true', () => {
    const appState = getAppState();
    const page = mountWithAppstate(appState, <Profile />);

    expect(page.find('.vertrokkenOnbekendWaarheen')).not.toBeNull();
    expect(page.find('.inOnderzoek')).not.toBeNull();
  });

  it('Displays an alert if the api responds with an error', () => {
    const appState = getAppState();
    appState.BRP.isError = true;
    appState.BRP.isDirty = true;

    const page = mountWithAppstate(appState, <Profile />);

    expect(page.find(`.InfoPanel.Verbintenis`)).toHaveLength(0);
    expect(page.find('[class*="Alert_Alert"]')).not.toBeNull();
  });
});
