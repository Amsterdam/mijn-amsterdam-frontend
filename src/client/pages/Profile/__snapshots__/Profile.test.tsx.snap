// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BRP Profile page Renders all the information provided 1`] = `"<main class=\\"Page DetailPage Profile\\"><header class=\\"PageHeading hasIcon\\"><span aria-hidden=\\"true\\" class=\\"Icon\\"><svg aria-label=\\"ROOT\\" fill=\\"black\\" class=\\"ChapterIcon\\">burgerzaken.svg</svg></span><div class=\\"HeadingInner\\"><h2 class=\\"Heading Heading__large\\">Mijn gegevens</h2></div></header><div class=\\"PageContent Intro\\"><p>In de Basisregistratie Personen legt de gemeente persoonsgegevens over u vast. Het gaat hier bijvoorbeeld om uw naam, adres, geboortedatum of uw burgerlijke staat. De gemeente gebruikt deze gegevens. Belangrijk dus dat deze gegevens kloppen.</p><div class=\\"Alert warning vertrokkenOnbekendWaarheen\\"><svg aria-hidden=\\"true\\" class=\\"Icon\\">Alert.svg</svg> <p>U staat sinds 01 januari 1967 in de BRP geregistreerd als \\"vertrokken – onbekend waarheen\\".</p><p>U kunt uw huidige adres doorgeven bij het Stadsloket. U moet hiervoor een <a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/veelgevraagd/?productid=%7BCAE578D9-A593-40FC-97C6-46BEA5B51319%7D\\" class=\\"Button Button__inline Button__lean Linkd LinkedInline\\">afspraak</a> maken.</p></div><div class=\\"Alert warning inOnderzoek\\"><svg aria-hidden=\\"true\\" class=\\"Icon\\">Alert.svg</svg> <p>Op dit moment onderzoeken wij of u nog steeds woont op het adres waar u ingeschreven staat.</p><p>U heeft hierover een brief ontvangen. Heeft u nog niet gereageerd? Voorkom een boete en stuur een bericht naar <a rel=\\"external noopener noreferrer\\" href=\\"mailto:adresonderzoek.basisinformatie@amsterdam.nl\\" class=\\"Button Button__inline Button__lean Linkd LinkedInline\\">adresonderzoek.basisinformatie@amsterdam.nl</a>.</p><p><a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/veelgevraagd/?productid=%7BCAE578D9-A593-40FC-97C6-46BEA5B51319%7D\\" class=\\"Button Button__inline Button__lean Linkd LinkedInline\\">Meer informatie</a>.</p></div></div><div class=\\"InfoPanel DefaultPanel\\"><h3 class=\\"Heading Heading__medium Title\\">Persoonlijke gegevens</h3><div class=\\"InfoPanelContent InfoPanelContent__persoonlijke-gegevens persoonlijke-gegevens\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Wesley</td></tr><tr class=\\"InfoPanelTableRow__titel\\"><th>Titel</th><td>Hertogin</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Beemsterboer</td></tr><tr class=\\"InfoPanelTableRow__geslacht\\"><th>Geslacht</th><td>Man</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>129095205</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>01 januari 1950</td></tr><tr class=\\"InfoPanelTableRow__geboorteplaats\\"><th>Geboorteplaats</th><td>Lochem</td></tr><tr class=\\"InfoPanelTableRow__geboorteland\\"><th>Geboorteland</th><td>Nederland</td></tr><tr class=\\"InfoPanelTableRow__nationaliteit\\"><th>Nationaliteit</th><td>Nederlandse Turkse</td></tr></tbody></table></div><ul class=\\"InfoPanelActionLinks\\"><li><a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/veelgevraagd/?productid=%7B989C04B3-AD81-4ABA-8DFE-465A29E2BF85%7D\\" class=\\"Button Button__plain Button__lean Linkd\\"><svg aria-hidden=\\"true\\" width=\\"14px\\" height=\\"14px\\" class=\\"Icon Icon__left\\">Chevron-Right.svg</svg>Inzien of correctie doorgeven</a></li></ul></div></div><div class=\\"InfoPanel DefaultPanel\\"><h3 class=\\"Heading Heading__medium Title\\">Woonadres</h3><div class=\\"InfoPanelContent InfoPanelContent__woonadres woonadres\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__straat\\"><th>Straat</th><td>Burgemeester Röellstr 1</td></tr><tr class=\\"InfoPanelTableRow__plaats\\"><th>Plaats</th><td>1064 BH Amsterdam</td></tr><tr class=\\"InfoPanelTableRow__vanaf\\"><th>Vanaf</th><td>01 januari 1967</td></tr></tbody></table></div><ul class=\\"InfoPanelActionLinks\\"><li><a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/burgerzaken/verhuizing-doorgeven/\\" class=\\"Button Button__plain Button__lean Linkd\\"><svg aria-hidden=\\"true\\" width=\\"14px\\" height=\\"14px\\" class=\\"Icon Icon__left\\">Chevron-Right.svg</svg>Verhuizing doorgeven</a></li></ul></div></div><section class=\\"SectionCollapsible\\"><h3 class=\\"Heading Heading__mediumLarge Title TitleWithItems\\"><button aria-expanded=\\"false\\" class=\\"TitleToggle\\"><svg aria-hidden=\\"true\\" class=\\"CaretIcon\\">Chevron-Right.svg</svg> Burgerlijke staat</button></h3><div class=\\"Panel\\" style=\\"height: 0px;\\" aria-hidden=\\"true\\"><div class=\\"PanelInner\\"><div id=\\"profile-verbintenis\\" class=\\"InfoPanel Verbintenis\\"><div class=\\"InfoPanelContent\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__status\\"><th>Status</th><td>Gehuwd</td></tr><tr class=\\"InfoPanelTableRow__vanaf\\"><th>Vanaf</th><td>01 januari 1999</td></tr><tr class=\\"InfoPanelTableRow__plaats\\"><th>Plaats</th><td>Asilah</td></tr><tr class=\\"InfoPanelTableRow__land\\"><th>Land</th><td>Marokko</td></tr><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Souad</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Bakker</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>08 juli 2019</td></tr></tbody></table></div><ul class=\\"InfoPanelActionLinks\\"><li><a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/veelgevraagd/?productid=%7B989C04B3-AD81-4ABA-8DFE-465A29E2BF85%7D\\" class=\\"Button Button__plain Button__lean Linkd\\"><svg aria-hidden=\\"true\\" width=\\"14px\\" height=\\"14px\\" class=\\"Icon Icon__left\\">Chevron-Right.svg</svg>Inzien of correctie doorgeven</a></li></ul></div></div></div></div></section><section class=\\"SectionCollapsible\\"><h3 class=\\"Heading Heading__mediumLarge Title TitleWithItems\\"><button aria-expanded=\\"false\\" class=\\"TitleToggle\\"><svg aria-hidden=\\"true\\" class=\\"CaretIcon\\">Chevron-Right.svg</svg> Voormalige verbintenissen</button></h3><div class=\\"Panel\\" style=\\"height: 0px;\\" aria-hidden=\\"true\\"><div class=\\"PanelInner\\"><div id=\\"profile-verbintenisHistorisch\\" class=\\"InfoPanel Verbintenis\\"><div class=\\"InfoPanelContent\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__verbintenis\\"><th>Verbintenis</th><td>Huwelijk</td></tr><tr class=\\"InfoPanelTableRow__vanaf\\"><th>Vanaf</th><td>01 januari 1912</td></tr><tr class=\\"InfoPanelTableRow__datum-ontbinding\\"><th>Datum ontbinding</th><td>08 juli 1998</td></tr><tr class=\\"InfoPanelTableRow__plaats\\"><th>Plaats</th><td>Asilah</td></tr><tr class=\\"InfoPanelTableRow__land\\"><th>Land</th><td>Marokko</td></tr><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Souad</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Bakker</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>08 juli 2019</td></tr></tbody></table><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__verbintenis\\"><th>Verbintenis</th><td>Huwelijk</td></tr><tr class=\\"InfoPanelTableRow__datum-ontbinding\\"><th>Datum ontbinding</th><td>08 juli 2019</td></tr><tr class=\\"InfoPanelTableRow__plaats\\"><th>Plaats</th><td>Asilah</td></tr><tr class=\\"InfoPanelTableRow__land\\"><th>Land</th><td>Marokko</td></tr><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Souad</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Bakker</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>08 juli 2019</td></tr></tbody></table></div><ul class=\\"InfoPanelActionLinks\\"><li><a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/veelgevraagd/?productid=%7B989C04B3-AD81-4ABA-8DFE-465A29E2BF85%7D\\" class=\\"Button Button__plain Button__lean Linkd\\"><svg aria-hidden=\\"true\\" width=\\"14px\\" height=\\"14px\\" class=\\"Icon Icon__left\\">Chevron-Right.svg</svg>Inzien of correctie doorgeven</a></li></ul></div></div></div></div></section><section class=\\"SectionCollapsible\\"><h3 class=\\"Heading Heading__mediumLarge Title TitleWithItems\\"><button aria-expanded=\\"false\\" class=\\"TitleToggle\\"><svg aria-hidden=\\"true\\" class=\\"CaretIcon\\">Chevron-Right.svg</svg> Kinderen</button></h3><div class=\\"Panel\\" style=\\"height: 0px;\\" aria-hidden=\\"true\\"><div class=\\"PanelInner\\"><div id=\\"profile-kinderen\\" class=\\"InfoPanel\\"><div class=\\"InfoPanelContent\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Yassine</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Kosterijk</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>08 juli 2019</td></tr></tbody></table><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Marwan</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Kosterijk</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>08 juli 2019</td></tr><tr class=\\"InfoPanelTableRow__datum-overlijden\\"><th>Datum overlijden</th><td>08 juli 2019</td></tr></tbody></table></div><ul class=\\"InfoPanelActionLinks\\"><li><a rel=\\"external noopener noreferrer\\" href=\\"https://www.amsterdam.nl/veelgevraagd/?productid=%7B989C04B3-AD81-4ABA-8DFE-465A29E2BF85%7D\\" class=\\"Button Button__plain Button__lean Linkd\\"><svg aria-hidden=\\"true\\" width=\\"14px\\" height=\\"14px\\" class=\\"Icon Icon__left\\">Chevron-Right.svg</svg>Inzien of correctie doorgeven</a></li></ul></div></div></div></div></section><section class=\\"SectionCollapsible\\"><h3 class=\\"Heading Heading__mediumLarge Title TitleWithItems\\"><button aria-expanded=\\"false\\" class=\\"TitleToggle\\"><svg aria-hidden=\\"true\\" class=\\"CaretIcon\\">Chevron-Right.svg</svg> Ouders</button></h3><div class=\\"Panel\\" style=\\"height: 0px;\\" aria-hidden=\\"true\\"><div class=\\"PanelInner\\"><div id=\\"profile-ouders\\" class=\\"InfoPanel\\"><div class=\\"InfoPanelContent\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Senior</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Beemsterboer</td></tr><tr class=\\"InfoPanelTableRow__geslacht\\"><th>Geslacht</th><td>Man</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>01 januari 1920</td></tr><tr class=\\"InfoPanelTableRow__geboorteplaats\\"><th>Geboorteplaats</th><td>Lochem</td></tr><tr class=\\"InfoPanelTableRow__geboorteland\\"><th>Geboorteland</th><td>Nederland</td></tr><tr class=\\"InfoPanelTableRow__nationaliteit\\"><th>Nationaliteit</th><td>Nederlandse</td></tr></tbody></table><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__voornamen\\"><th>Voornamen</th><td>Seniora</td></tr><tr class=\\"InfoPanelTableRow__achternaam\\"><th>Achternaam</th><td>Beemsterboerin</td></tr><tr class=\\"InfoPanelTableRow__geslacht\\"><th>Geslacht</th><td>Vrouw</td></tr><tr class=\\"InfoPanelTableRow__bsn\\"><th>BSN</th><td>123456780</td></tr><tr class=\\"InfoPanelTableRow__geboortedatum\\"><th>Geboortedatum</th><td>01 januari 1920</td></tr><tr class=\\"InfoPanelTableRow__geboorteplaats\\"><th>Geboorteplaats</th><td>Lochem</td></tr><tr class=\\"InfoPanelTableRow__geboorteland\\"><th>Geboorteland</th><td>Nederland</td></tr><tr class=\\"InfoPanelTableRow__nationaliteit\\"><th>Nationaliteit</th><td>Nederlandse</td></tr></tbody></table></div></div></div></div></div></section><section class=\\"SectionCollapsible\\"><h3 class=\\"Heading Heading__mediumLarge Title TitleWithItems\\"><button aria-expanded=\\"false\\" class=\\"TitleToggle\\"><svg aria-hidden=\\"true\\" class=\\"CaretIcon\\">Chevron-Right.svg</svg> Vorige woonadressen</button></h3><div class=\\"Panel\\" style=\\"height: 0px;\\" aria-hidden=\\"true\\"><div class=\\"PanelInner\\"><div id=\\"profile-adresHistorisch\\" class=\\"InfoPanel\\"><div class=\\"InfoPanelContent\\"><div class=\\"TableWrap\\"><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__straat\\"><th>Straat</th><td>Burgemeester Röellstr 1</td></tr><tr class=\\"InfoPanelTableRow__plaats\\"><th>Plaats</th><td>1064 BH Amsterdam</td></tr><tr class=\\"InfoPanelTableRow__vanaf\\"><th>Vanaf</th><td>01 januari 1967</td></tr></tbody></table><table class=\\"InfoPanelTable\\"><tbody><tr class=\\"InfoPanelTableRow__straat\\"><th>Straat</th><td>Burgemeester Röellstr 1</td></tr><tr class=\\"InfoPanelTableRow__plaats\\"><th>Plaats</th><td>1064 BH Amsterdam</td></tr><tr class=\\"InfoPanelTableRow__vanaf\\"><th>Vanaf</th><td>01 januari 1967</td></tr></tbody></table></div></div></div></div></div></section></main>"`;
